src/CoreBundle/Util/FieldTranslationUtil.php

```diff
@@ -222,7 +222,9 @@ private function changeTranslatableState($fieldId, $translatable)
 
         $translatableString = $translatable ? '1' : '0';
         $tableEditorManager = TTools::GetTableEditorManager('cms_field_conf', $fieldId);
+        $tableEditorManager->AllowEditByAll(true);
```

Comment:
This wasn't a problem before?

Reply:
It was, or at least I can't imagine that it wasn't a problem. So maybe there a systems where fields were not configured translatable.